Cargo.lock

```diff
@@ -803,28 +799,28 @@ dependencies = [
 
 [[package]]
 name = "num_cpus"
-version = "1.13.1"
+version = "1.15.0"
 source = "registry+https://github.com/rust-lang/crates.io-index"
-checksum = "19e64526ebdee182341572e50e9ad03965aa510cd94427a4549448f285e957a1"
+checksum = "0fac9e2da13b5eb447a6ce3d392f23a29d8694bff781bf03a16cd9ac8697593b"
 dependencies = [
- "hermit-abi",
+ "hermit-abi 0.2.6",
```

Comment:
This isn't really ideal.  It's allowed, because we're only checking deps for linux/x64, but they still need to get downloaded and handled by bazel and so on.  Can't really do anything about it but good to keep an eye on it in case there are future opportunities to update `num_cpus` to a version that uses 0.3+ (they've updated  in their main branch but haven't published).  Not a big deal.